src/Microsoft.OData.Core/UriParser/Aggregation/AggregationMethod.cs

```diff
@@ -9,7 +9,31 @@ namespace Microsoft.OData.UriParser.Aggregation
     /// <summary>
     /// Enumeration of methods used in the aggregation clause
     /// </summary>
-    public enum AggregationMethod
+    public sealed class AggregationMethod

```

Comment:
To avoid a breaking API change, lets make this "AggregationMethodDefinition", leave AggregationMethod as-is but add "Custom".   Rename MethodType to MethodKind please.